Binary files upstream.original/.git/index and upstream.patched/.git/index differ
diff -rupN upstream.original/JASidePanels/Source/UIViewController+JASidePanel.h upstream.patched/JASidePanels/Source/UIViewController+JASidePanel.h
--- upstream.original/JASidePanels/Source/UIViewController+JASidePanel.h	2013-08-20 11:46:29.000000000 +0300
+++ upstream.patched/JASidePanels/Source/UIViewController+JASidePanel.h	2013-08-20 11:50:25.000000000 +0300
@@ -24,6 +24,7 @@
  */
 
 #import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
 
 @class JASidePanelController;
 
